docs/sources/next/using-k6/k6-options/reference.md

```diff
@@ -352,13 +352,13 @@ export const options = {
 
 {{< /code >}}
 
-## Extension options
+## Cloud options
 
 An object used to set configuration options for cloud parameters and third-party collectors, like plugins. For more information about available parameters, refer to [Cloud options](https://grafana.com/docs/grafana-cloud/k6/author-run/cloud-scripting-extras/cloud-options/).
```

Comment:
@olegbespalov, I saw you removed "third-party collectors" from the table above. Do we want to remove it from here as well?

Reply:
Yep, it makes sense to remove this from here, good catch :+1:  I'll open a PR